framework/core/src/User/AvatarUploader.php

```diff
@@ -13,12 +13,13 @@
 use Illuminate\Contracts\Filesystem\Filesystem;
 use Illuminate\Support\Str;
 use Intervention\Image\Image;
+use Intervention\Image\ImageManager;
 
 class AvatarUploader
 {
     protected Filesystem $uploadDir;
 
-    public function __construct(Factory $filesystemFactory)
+    public function __construct(Factory $filesystemFactory, protected ImageManager $imageManager)
```

Comment:
why was this added? seems unused

Reply:
Probably a hangover from where I attempted to upgrade `intervention/image` on this branch 🙈 I'll check it out and revert